defaults/main.yml

```diff
@@ -32,3 +32,7 @@ ovirt_engine_setup_perform_upgrade: false
 
 ovirt_engine_apache_config_ssl: true
 ovirt_engine_apache_config_root_redirection: true
+
+restore_cmd: 'engine-backup --mode=restore'
```

Comment:
maybe create `vars` folder with `main.yml` we don't want the user to change the restore_cmd

Reply:
created and removed from defaults